### DISCUSSION TITLE
convert yolov5s model to onnx and run it in onnx runtime

### DISCUSSION THREAD

**Discussion by pk-1196** [ultralytics/yolov5#3279] [General]
Hii can anyone provide me the steps to run yolov5s model in onnx runtime.
I have trained a yolo model on my own data using yolov5s.yaml and saved the weight file as best.pt 
Which is showing good detection but want to run in onnx runtime.
Any help will we appreciated .Thanks

**Comment by benjamingriff** [comment]
@pk-1196 Did you manage to achieve this? I am trying to do the same thing.

**Comment by mattpopovich** [comment]
Are you aware of the ONNX guide? 
https://github.com/ultralytics/yolov5/issues/251
